src/lib/storage/index/partial_hash/partial_hash_index.cpp

```diff
@@ -12,36 +12,30 @@ size_t PartialHashIndex::estimate_memory_consumption(const ChunkOffset row_count
 PartialHashIndex::PartialHashIndex(const std::vector<std::pair<ChunkID, std::shared_ptr<Chunk>>>& chunks_to_index,
                                    const ColumnID column_id)
     : AbstractTableIndex{get_table_index_type_of<PartialHashIndex>()}, _column_id(column_id) {
-  if (!chunks_to_index.empty()) {
-    insert_entries(chunks_to_index);
-  } else {
-    /**
-     * Because 'chunks_to_index' is empty, we cannot determine the data type of the column and therefore construct
-     * an empty impl object. When chunks are added to this index, it is swapped out again.
-     */
-    _impl = std::make_shared<BasePartialHashIndexImpl>();
```

Comment:
@Bouncner this was the code location where we created the BasePartialHashIndexImpl since `chunk_to_index` is empty.

Reply:
I think, I got it know. :D